app/services/twitter_service.rb

```diff
@@ -0,0 +1,13 @@
+class TwitterService
+  APP_CONFIG = YAML.load(File.read('config/settings/development.yml'))
```

Comment:
Guess you are using this wrong.... the point of using a settings gem is not to load it manually. And especially have it load the right constant for the right environment. Here you hardcode development.yml